pkgs/servers/monitoring/grafana/plugins/bigquery-grafana/default.nix

```diff
@@ -0,0 +1,13 @@
+{ grafanaPlugin, lib }:
+
+grafanaPlugin rec {
+  pname = "doitintl-bigquery-datasource";
```

Comment:
I would prefer if the pname and the attribute name match - can you change the attribute name to be "doitintl-bigquery-datasource" (and rename the directory)?

Reply:
Thanks for the feedback. Changed commit message, attr name and dir name.